src/classes/TDTM_Runnable.cls

```diff
@@ -57,63 +57,63 @@ global abstract class TDTM_Runnable {
      * @description Calls the run method of the class, as a future. To be called dynamically from TDTM_TriggerHandler. 
      * We need this method because Apex won't let us call a non-static method from a separate object instance.
      * @param newIds The IDs of the records that were passed to the trigger as trigger.new.
-     * @param oldIds The IDs of the records that were passed to the trigger as trigger.old.
+     * @param serializedOldList The list of serialized records that were passed to the trigger as trigger.old.
      * @param action The event (before insert, after insert,...) that caused the class to run.
      * @param objectName The name of the SObject the class runs for.
      * @param classToRunName The name of the class to run.
      * @return void
      */
-    public void runFutureNonStatic(Set<Id> newIds, Set<Id> oldIds, String action, String objectName, String classToRunName) {
-        runFuture(newIds, oldIds, action, objectName, classToRunName);
+    public void runFutureNonStatic(Set<Id> newIds, List<String> serializedOldList, String action, String objectName, String classToRunName) {
+        runFuture(newIds, serializedOldList, action, objectName, classToRunName);
     }
     
     /*******************************************************************************************************
      * @description Calls the run method of the class, as a future.
      * @param newIds The IDs of the records that were passed to the trigger as trigger.new.
-     * @param oldIds The IDs of the records that were passed to the trigger as trigger.old.
+     * @param serializedOldList The list of serialized records that were passed to the trigger as trigger.old..
      * @param action The event (before insert, after insert,...) that caused the class to run.
      * @param objectName The name of the SObject the class runs for.
      * @param classToRunName The name of the class to run.
      * @return void
      */
     @future
-    public static void runFuture(Set<Id> newIds, Set<Id> oldIds, String action, String objectName, String classToRunName) {
-        
+    public static void runFuture(Set<Id> newIds, List<String> serializedOldList, String action, String objectName, String classToRunName) {
+
         Schema.DescribeSObjectResult objResult = UTIL_Describe.getObjectDescribe(objectName);
         TDTM_Runnable.Action triggerAction = TDTM_TriggerActionHelper.getTriggerActionFromName(action);
-        
+
         //Get all fields for the object
         List<Schema.SObjectField> allFields = objResult.fields.getMap().values();
-        
+
         //Getting the records the IDs refer to
-        String qn = 'SELECT ';
-        for (Integer i = 0; i < allFields.size() - 1; i++) { //add all fields, except last, to query
+        String qn = 'select ';
```

Comment:
@manikrai29 Thank you for the submission. Very much appreciated. I haven't had a chance to look at the bulk of this PR, however I did notice that you undid our standard styling by changing SOQL to lower case, renaming vars, removing spaces, removing curly brackets, etc.. Can you please revert the style changes and limit the PR just just functional changes that address the issue? That will help us by not having to put these back as well as reduce the review time once one of our Scrum teams is able to pull this in. Thank you.